temporal/t.rast.aggregate/t.rast.aggregate.py

```diff
@@ -128,13 +133,17 @@ def main():
     gran = options["granularity"]
     base = options["basename"]
     register_null = flags["n"]
+    weighting = flags["w"]
     method = options["method"]
     sampling = options["sampling"]
     offset = options["offset"]
     nprocs = options["nprocs"]
     file_limit = options["file_limit"]
     time_suffix = options["suffix"]
 
+    if weighting and not sampling == "related":
+        gcore.fatal(_("Weighting only works with method 'related'"))
```

Comment:
Confusing: This says "method", but the name of the option referred to is "sampling".